charts/ks-devops/charts/jenkins/README.md

```diff
@@ -83,17 +83,18 @@ The following tables list the configurable parameters of the Jenkins chart and t
-| `Agent.ImageTag`           | Agent image tag                                 | `2.62`                 |
-| `Agent.Privileged`         | Agent privileged container                      | `false`                |
-| `Agent.resources`          | Resources allocation (Requests and Limits)      | `{requests: {cpu: 200m, memory: 256Mi}, limits: {cpu: 200m, memory: 256Mi}}`|
-| `Agent.volumes`            | Additional volumes                              | `nil`                  |
+| Parameter                   | Description                                                                                        | Default                                                                      |
+| --------------------------- | -------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------- |
+| `Agent.AlwaysPullImage`     | Always pull agent container image before build                                                     | `false`                                                                      |
+| `Agent.CustomJenkinsLabels` | Append Jenkins labels to the agent                                                                 | `{}`                                                                         |
+| `Agent.Enabled`             | Enable Kubernetes plugin jnlp-agent podTemplate                                                    | `true`                                                                       |
+| `Agent.Image`               | Agent image name                                                                                   | `jenkinsci/jnlp-slave`                                                       |
+| `Agent.ImagePullSecret`     | Agent image pull secret                                                                            | Not set                                                                      |
+| `Agent.ImageTag`            | Agent image tag                                                                                    | `2.62`                                                                       |
+| `Agent.Privileged`          | Agent privileged container                                                                         | `false`                                                                      |
+| `Agent.resources`           | Resources allocation (Requests and Limits)                                                         | `{requests: {cpu: 200m, memory: 256Mi}, limits: {cpu: 200m, memory: 256Mi}}` |
+| `Agent.volumes`             | Additional volumes                                                                                 | `nil`                                                                        |
+| `Agent.Builder.Registry`    | Agent builder image registry with namespace. Like `ghcr.io/kubesphere` or `[docker.io/]kubesphere` | `kubespheredev`
```

Comment:
I know the prefix `docker.io` is optional if the container runtime is Docker. But I prefer to use `docker.io/kubesphere`. There're two pros:

* There's no potential confusion for anyone
  * Some users might don't understand the meaning of `[docker.io/]kubesphere`
* Other container runtime does not take `docker.io` as the default prefix, such as containerd

Reply:
Thanks for you suggestion. I will take the advice.